src/components/faq/faqComponent.jsx

```diff
@@ -55,7 +55,7 @@ const FaqComponent = () => (
             <Col xs={12} id="scorecard">
                 <h4>What do the scores mean?</h4>
                 <Well>
-                    <div dangerouslySetInnerHTML={createScorecard()}></div>
+                    <a href="assets/d4d85ae49e726dcf637e2eacb3651afc.md">View Read Me</a>
```

Comment:
So I'mm assuming that hash will change everytime we serve up the app... I think ideally we'd like to have it work similarly, where we actually read into the markdown and render in the app.